blocks/inputClub.js

```diff
@@ -1,5 +1,11 @@
 // ブロックキットを使用しステップ設定モーダルのUIを構築
-module.exports = [
+const clubName = '部活動名';
+const description = '活動内容';
+const captain = '部長名';
+const collaborator1 = '初期メンバー1';
+const collaborator2 = '初期メンバー2';
+
+const inputClub = [
```

Comment:
```suggestion
const inputClubModal = [
```
などのように file name で既に inputClub.js と銘打ってるんで、モーダルなどの実際に何をする変数なのかを定義した方が export, inport した際に（module, require も一緒）わかりやすいので僕は好きです
e.g. inputClubModal, modal, form, modalInputClub .....

Reply:
なるほど！
修正します！